### DISCUSSION TITLE
About the required specifications of WinMerge

### DISCUSSION THREAD

**Discussion by fj0538gu** [WinMerge/winmerge#2358] [Q&A] [answered]
I am currently summarizing the operating conditions for the software.
Please tell me the minimum and recommended CPU model required for WinMerge to work.

現在、ソフトの要求スペックについてまとめています。
WinMergeが動作するのに必要なCPUモデルの最低値と推奨値を教えて下さい。

**Comment by sdottaka** [comment, marked as answer]
The 32-bit version of WinMerge operates on Windows XP SP3 or later. Since it uses the SSE instruction set, a Pentium III or higher CPU is required.

The 64-bit version of WinMerge operates on Windows 7 or later. Since it uses the SSE2 instruction set, a Pentium 4 or higher CPU is required.

When comparing two files, if the content is nearly identical, having available memory larger than twice the combined size of the two files should suffice. If the files are completely different, the amount of memory used will depend on the number of lines in the files being compared.

When comparing folders, the memory usage will depend on the number of files within the folders.

Whether WinMerge runs smoothly on your PC depends on the amount and number of files being compared. Please test it on your system to ensure satisfactory performance.

**Reply by fj0538gu** [reply]
well understood. Thank you for answering!
